cpp/src/arrow/adapters/arrow-rados-cls/README.md

```diff
@@ -76,3 +76,7 @@ docker-compose run --service-ports ubuntu-cls-demo
 # Development
 
 Check out these [instructions](docs/contributing.md) for setting up a local development environment.
+
+# Publication
+
+[Towards an Arrow-native Storage System](https://arxiv.org/pdf/2105.09894.pdf)
```

Comment:
Can we put it in a numbered list ?

Reply:
But there is just one, so a list would be unnecessary in my opinion. When we add more publications, maybe then a list would be more appropriate, don't you think?

Reply:
We are about to add more actually, like from our collaborators from Leiden univ.

Reply:
Also rename to `Publications` (prural)

Reply:
Done.